src/client/logging/levels.ts

```diff
@@ -9,14 +9,16 @@ import * as winston from 'winston';
 
 export enum LogLevel {
     // Larger numbers are higher priority.
+    Off = 50,
     Error = 40,
     Warn = 30,
     Info = 20,
     Debug = 10,
     Trace = 5
```

Comment:
I recommend making it clear that "Off" is special:

```suggestion
    Error = 40,
    Warn = 30,
    Info = 20,
    Debug = 10,
    Trace = 5,
    // "Off" means "log nothing".
    Off = 0
```

Note that convention of making "off" have a value of 0.  This is what Python's logging module does.

Reply:
> Note that convention of making "off" have a value of 0. This is what Python's logging module does.

Lower priority means we're logging more stuff, and higher the priority the less things we're logging, which is why it made sense to assign `Off` the highest priority.